Makefile

```diff
@@ -13,7 +13,7 @@ run:
 	java -Xmx2G -server -XX:+UseG1GC -XX:MaxGCPauseMillis=20 -XX:InitiatingHeapOccupancyPercent=35 -XX:+ExplicitGCInvokesConcurrent -Djava.awt.headless=true -jar assembly/target/stream-registry*SNAPSHOT.jar server config-dev.yaml
 
 debug:
-	java -Xmx2G -server -XX:+UseG1GC -XX:MaxGCPauseMillis=20 -XX:InitiatingHeapOccupancyPercent=35 -XX:+ExplicitGCInvokesConcurrent -Djava.awt.headless=true -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5105 -jar assembly/target/stream-registry*SNAPSHOT.jar server config-dev.yaml
+	java -Xmx2G -server -XX:+UseG1GC -XX:MaxGCPauseMillis=20 -XX:InitiatingHeapOccupancyPercent=35 -XX:+ExplicitGCInvokesConcurrent -Djava.awt.headless=true -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5105 -jar assembly/target/stream-registry*SNAPSHOT.jar server config-dev.yaml
```

Comment:
Suggest a change to `suspend=${SCHEMA_REGISTRY_DEBUG_SUSPEND:-'n'}`

Reply:
yeah. something like that ^^^ would be great!

Reply:
modified so that suspend is 'n' by default but running `STREAM_REGISTRY_DEBUG_SUSPEND=y make debug` will suspend